README.md

```diff
@@ -102,7 +102,7 @@ Then open settings screen -> developer and paste
 Connect to the 
 - Hacknet: `wss://hack.hydradx.io:9944` 
 - [Stakenet](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc-01.snakenet.hydradx.io): `wss://rpc-01.snakenet.hydradx.io`
-- or local node.
+- or local node – if you are on brave / chrome, set chrome://flags/#allow-insecure-localhost
```

Comment:
nitpick: I believe you can just mentioned `if you are using chromium based browser` ? so it would apply to all of them